case-lib/hijack.sh

```diff
@@ -25,11 +25,6 @@ function func_exit_handler()
 
     # when sof logger collect is open
     if [ "X$SOF_LOG_COLLECT" == "X1" ]; then
-        # when error occurs, exit and catch etrace log
-        [[ $exit_status -eq 1 ]] && {
-            func_lib_start_log_collect 1
-            sleep 1s
-        }
```

Comment:
@marc-hb Not sure about this. This is not the normal logger run, but capture the separate etrace log in case of output (etrace.txt). Is this already covered somewhere else?

Reply:
Thanks @kv2019i for the catch, really appreciated. I've been looking for information about the difference between `debug/trace` and `debug/etrace` but didn't find much, any good reference? @ktrzcinx ? I didn't expect both could be used simultaneously, in fact I now realize this PR is (was) an attempt to avoid a failure to use both simultaneously.

Reply:
Found something here: https://github.com/thesofproject/sof/issues/3265#issuecomment-669802163
> thanks a lot for your information about traces. This is the kind of information we need to write it in sof-docs.